### ISSUE TITLE
Bisq notifications UI

### ISSUE THREAD

**Issue by ManfredKarrer** [bisq-network/proposals#29]
Additional to the proposal for the Notification [mobile app](https://github.com/bisq-network/proposals/issues/25) we need a concept and UI design for setting the notifications inside Bisq.
This draft should give only the very basic idea and overview in the hope that any contributor can work on that to work on a more detailed proposal.

## Overview:
There should be at least those types of notifications:
- Notification when a user's open offer is taken
- Notification for each trade step (tx confirmation, payment started, payment received, payout done)
- Offer alert: Set a price range for offers with a specific currency and payment method when you want to get notified 

Nice to have:
- Prices alert: Set price range for currency when you want to get a notification
- Not a notification but a similar message would be to send a "wipe out" command to the mobile to delete all app data (encryption key, notifications) remotely for the case the mobile gets stolen.

### Setup:
For the one-time setup (key and mobile token exchange from mobile app to Bisq app by scanning a QR code) we need a screen as well. That can be added to the Account menu as additional sub item.
It would contain only a button for scanning the QR code with the computers camera, a input text field for those who have no camera (key and mobile token is sent by email to computer) and a display field for the key and token.
We only support one mobile app to be linked to one Bisq app.

### Notifications:
For setting up and editing the notifications we need a new screen with sufficient space and several submenues. It should be extendable to add new notification types in future.
We are limited with the available space at the top menu (a BSQ icon will be added as well once the BSQ-DAO is complete). So we should try to get it into an existing menu item or think of ways to combine existing ones (without creating tom much dev effort). Maybe the notifications don't need a standard icon but some different styled small icon?
We could also integrate a icon button to all relevant screens (e.g. open offers, offer book,...) to link to the notification screen to make it easier to users to find it. The main menu could be then inside of settings (new tab). When we launch the new feature we can add a one-time popup with pointing the user to the new feature.

So what is mainly needed is a good concept where and how to place and link the notifications screen as well as to make rough layouts for the different setup/edit notifications screens. That can be wire frame layouts. For that task no development skills are required.

Design-wise we would stick with the current UI style atm (will be reskinned with rest of app soon).

**Comment by ManfredKarrer** [comment]
See update at: https://github.com/bisq-network/proposals/issues/25#issuecomment-405132572

**Comment by cbeams** [comment]
This proposal never got any reactions or feedback, and probably should have been dealt with in another forum. The main question with the proposal at #25 was "should a Bisq Remote app be built, and would we support it from the Bisq Desktop app if it were". Once the answer to that was clear, details about how the UI should be implemeted, etc probably shouldn't try to be managed as proposals per se, but rather in discussion threads somewhere. It bugs me that we still don't really have a good place to do this (Slack is too ephemeral for important decisions). I'm open to suggestions. But in any case, we should just close this proposal, because the work it describes has already been done.

**Comment by ManfredKarrer** [comment]
Yes agree. An issue would have been probably a better choice.
